### ISSUE TITLE
Webpack Compilation Error ERR_REQUIRE_ESM

### ISSUE THREAD

**Issue by anark** [cypress-io/cypress#9703]
Hi there,
I am getting a strange error when running my unit tests after upgrading some other packages.  I am running webpack via the `loadWebpack` plugin and getting an error that I do not get when running my app in any other way(I only get this error when trying to compile the unit tests)

```
Webpack Compilation Error
./app/javascript/components/ComboBox/ComboBox.tsx
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: ./node_modules/colorette/index.js
require() of ES modules is not supported.
require() of ./node_modules/colorette/index.js from ./node_modules/postcss/lib/css-syntax-error.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from ./node_modules/colorette/package.json.
```

My cypress/plugins/index.js file is as follows
```js
module.exports = (on, config) => {
  process.env.RAILS_ENV = 'cypress'
  process.env.NODE_ENV = 'development'

  config.env.webpackFilename = 'config/webpack/development.js'
  require('cypress-react-unit-test/plugins/load-webpack')(on, config)

  return config
}
```

Any help would be greatly appreciated.  It seems to have started after updating tailwind to the newest version but having trouble diagnosing.  I do not explicitly use the colorette package but some other package may be.  The funny thing is that this doesn't happen when running via webpack CLI, and when I investigate `node_modules/colorette` it does in fact have an index.cjs file.

**Comment by anark** [comment]
I am getting this same error in a separate project after upgrading to the latest postcss in that project.

Does anyone have any ideas as to what to try to get around this error.  Reading up it sounds like this can happen if node is running in version 13, however on my system I am running node 14 and have tried with and withou setting `nodeVersion` in my cypress.json to `system` .

Any help is greatly appreciated.

**Comment by dmtrKovalenko** [comment]
This error is coming from our hardcoded cjs babel injection ([this line](https://github.com/cypress-io/cypress-react-unit-test/blob/larrifax/main/plugins/load-webpack/index.js#L36) – already removed and will be released as a part of 7.0

**Comment by lmiller1990** [comment]
Hey @anark,

We just released Cypress 7.0, which comes with a completely revamped Component Testing experience. Component testing is no longer experimental (but still in alpha, with lots of updates and improvements coming in the next few weeks).

- Updated Component Testing intro: https://docs.cypress.io/guides/component-testing/introduction
- Changelog: https://docs.cypress.io/guides/references/changelog
- Migration guide: https://docs.cypress.io/guides/references/migration-guide#Component-Testing

We now have a dedicated component testing runner (which builds on top of the existing runner, but has some changes like not needing to launch the electron app to select a spec, and using a dev-server architecture instead of file:preprocessor, making it a (lot) faster.

This release includes the fix mentioned by @dmtrKovalenko. Can you please upgrade (see migration guide) and see if this fixes your problem?

Thanks! Let us know if you still have the problem.

**Comment by anark** [comment]
Thanks @lmiller1990,
My temporary fix was to set `TS_NODE_PROJECT=0` but it seems that is no longer needed in cypress 7.0.  Thanks for the fix!
